src/i18n/en.json

```diff
@@ -0,0 +1,11 @@
+{
+	"login": {
+		"title": "Welcome to\n",
+		"growName": "Grow Hub!",
+		"subTitle": "Your pocket guide for plant care",
+		"linkText": "Log in as a guest"
+	},
+	"components": {
+		"authBotton": "Log in with"
+	}
```

Comment:
I would strongly recommend against nested vocabularies. They are harder to maintain and its way easier to create translation duplications in nested vocabularies